__mocks__/@loadable/component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+function loadable(load: any, { resolveComponent }) {
+  const Component = resolveComponent(load.requireSync())
+
+  const Loadable = (props: any) => {
+    return <Component {...props} />
+  }
+  return Loadable
+}
+
+export default loadable
```

Comment:
We needed a way to test bundle split components. So this rewires things so that any time a bundle split component is seen it loads its synchronous counterpart